src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public abstract class Figure
+        implements FigureArea, FigureDrow {
+    private String color;
+    private String figureType;
```

Comment:
```suggestion
```

Reply:
The figureType value is used in several places in the code, it would be better to keep it in one place. Each Figure has its own type name, why  we can't store this in the corresponding attribute?

Reply:
You don't need to create or use a figure type anywhere, and if you have questions about the review, ask them at slack